sdks/java/io/rrio/build.gradle

```diff
@@ -37,6 +37,8 @@ dependencies {
     implementation library.java.jackson_core
     implementation library.java.jackson_databind
     implementation "redis.clients:jedis:$jedisVersion"
+    implementation platform(library.java.google_cloud_platform_libraries_bom)
```

Comment:
It now depends on google_cloud_platform_libraries_bom because it uses `com.google.api.client.util.BackOff` and `ExponentialBackOff`. This use alone does not necessarily need to introduce the whole gcp_bom dependency. Consider use beam's `org.apache.beam.sdk.util.BackOff`

In general I understand RRIO as generic for API calls and should not need to depend on GCP